recipes-extended/xen/files/defconfig

```diff
@@ -29,19 +42,28 @@ CONFIG_GRANT_TABLE=y
 # CONFIG_GRANT_TABLE_EXOTIC is not set
 CONFIG_HAS_ALTERNATIVE=y
 CONFIG_HAS_EX_TABLE=y
+CONFIG_HAS_FAST_MULTIPLY=y
 # CONFIG_MEM_ACCESS_ALWAYS_ON is not set
 # CONFIG_MEM_ACCESS is not set
-# CONFIG_HAS_MEM_PAGING is not set
-# CONFIG_HAS_MEM_SHARING is not set
+CONFIG_HAS_MEM_PAGING=y
```

Comment:
This one stands out to me since it seems to apply to HVM guests, what do we need it for? Or better question, what is it useful for?

Reply:
The "*HAS*" ones like CONFIG_HAS_MEM_PAGING just means the architecture has support for the feature - not that is it used/enabled.

Oh, defconfig might need a refresh for 4.16 since I don't see it down in the 4.16 build directory.

Reply:
I think it might just be
```
CONFIG_MEM_PAGING
```
now.

Reply:
The .config down in the build directory has:
`# CONFIG_MEM_PAGING is not set`

I'll add a commit refreshing the defconfig for 4.16.